lib/chef/provider/mount/aix.rb

```diff
@@ -28,7 +28,7 @@ def initialize(new_resource, run_context)
           super
           # options and fstype are set to "defaults" and "auto" respectively in the Mount Resource class. These options are not valid for AIX, override them.
           if @new_resource.options[0] == "defaults"
-            @new_resource.options.clear
+            @new_resource.options([])
```

Comment:
Is there a reason not to just set these correctly for AIX in the resource?

Reply:
wild spaghetti code in the resource v twiddling stuff in the provider...

Reply:
I mean, i think either is bad, but i think this is marginally less bad

Reply:
fair enough